boards/st/steval_stwinbx1/steval_stwinbx1.dts

```diff
@@ -0,0 +1,288 @@
+/*
+ * Copyright (c) 2024 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+ #include <st/u5/stm32u585Xi.dtsi>
+ #include <st/u5/stm32u585aiixq-pinctrl.dtsi>
+ #include <zephyr/dt-bindings/input/input-event-codes.h>
+
+/ {
+	model = "STMicroelectronics STEVAL-STWINBX1 Development kit";
+	compatible = "st,steval_stwinbx1";
+
```

Comment:
@HoZHel, could you please post a suggestion to enable BLE? I think there’s a BlueNRG-M2 module in this board connected over SPI.

Reply:
@HoZhel, as a second thought, after talking with Erwan, I think it would be cleaner to have a separate PR.